save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/CommentDto.kt

```diff
@@ -13,13 +13,15 @@ data class CommentDto(
     val message: String,
     val userName: String,
     val userAvatar: String?,
+    val createDate: Long?,
```

Comment:
Suggestion to use LocalDateTime instead of it

Reply:
Fixed